### DISCUSSION TITLE
Is it possible to display the selected node's full path from root, when a node is selected.

### DISCUSSION THREAD

**Discussion by maria88ahmed** [dowjones/react-dropdown-tree-select#535] [Q&A]
I want to display the full path rather than just the node label , in the case of mode='radioSelect'. 

For eg: to display Parent1 > Child1 -> Child12 instead of Child12 , when Child12 is selected

**Comment by sinhpn92** [comment]
Did you solve this? @maria88ahmed

**Comment by maria88ahmed** [comment]
Yes, using HOC

**Comment by sinhpn92** [comment]
Ok thank you so   much @maria88ahmed

**Reply by munishchoudhary204** [reply]
how please tell me give some code
